src/main/resources/org/schemaspy/types/mssql05.properties

```diff
@@ -24,7 +24,7 @@ dbThreads=1
 
 # return table_name, comments for current schema
 # SQL provided by Stefano Santoro
-selectTableCommentsSql=SELECT i_s.TABLE_NAME, CAST(s.value AS VARCHAR(MAX)) AS comments FROM INFORMATION_SCHEMA.Tables i_s INNER JOIN sys.extended_properties s ON s.major_id \= OBJECT_ID(i_s.table_catalog + '..' + i_s.table_name) WHERE s.class \= 1 AND s.name \= 'MS_Description' AND s.minor_id \= 0
+selectTableCommentsSql=SELECT i_s.TABLE_NAME, CAST(s.value AS VARCHAR(MAX)) AS comments FROM INFORMATION_SCHEMA.Tables i_s INNER JOIN sys.extended_properties s ON s.major_id \= OBJECT_ID(i_s.table_catalog + '.' + i_s.table_schema + '.' + i_s.table_name) WHERE s.class \= 1 AND s.name \= 'MS_Description' AND s.minor_id \= 0
```

Comment:
need to append `AND i_s.table_schema = :schema` to end of query to make table comments test in #325 to pass.